front-endd/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,14 @@
+// added routes
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
-
-const routes: Routes = [];
+import { StudendcrudComponent } from './studendcrud/studendcrud.component';
+const routes: Routes = [
+  { path: 'users/new', component: StudendcrudComponent },
+  { path: 'users/:id/edit', component: StudendcrudComponent },
+  { path: 'users/:id', component: StudendcrudComponent },
+  { path: 'users', component: StudendcrudComponent },
```

Comment:
why the path name is users? Please rename it to students as we are dealing with student records here